app/views/bookings/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container py-5">
+  <div class="row">
+    <div class="col-12 col-md-6 offset-lg-3">
+      <div class="bg-white rounded p-3 shadow">
+        <h3>Add your Game here:</h3>
+        <%= simple_form_for [current_user, @booking] do |f| %>
+          <%= f.input :category, placeholder: 'Pinball' %>
```

Comment:
change with the Game::CATEGORY constant to have a select menu with only decided category